src/main/java/Cleese/TaskList.java

```diff
@@ -0,0 +1,35 @@
+package Cleese;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private final ArrayList<Task> taskList;
```

Comment:
The name of the field can be called tasks instead as it can be confused with the class name and be prone to mistakes referring to the class and object.